test/integration/start_stop_delete_test.go

```diff
@@ -63,17 +64,18 @@ func TestStartStop(t *testing.T) {
 		}},
 	}
 
-	for _, test := range tests {
-		t.Run(test.name, func(t *testing.T) {
-			r := NewMinikubeRunner(t)
-			if !strings.Contains(test.name, "docker") && usingNoneDriver(r) {
-				t.Skipf("skipping %s - incompatible with none driver", test.name)
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			testName := tc.name
```

Comment:
Just use `tc.name` rather than introducing a second variable representing the test name.